.pre-commit-hooks.yaml

```
@@ -5,5 +5,5 @@
   language: python
   language_version: python3
   pass_filenames: false
-  files: ^poetry.lock$
+  files: ^poetry\.lock$
```

```suggestion
  files: ^(.*/)?poetry\.lock$
```

Let's keep it in sync with poetry's hooks.

---

Good idea, thanks! Applied.